tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -2,16 +2,51 @@
 
 from .models import ParserErrorCategoryChoices
 from .util import get_rpt_month_year_list
+from tdpservice.parsers.schema_defs.utils import get_program_model
 import logging
 
 logger = logging.getLogger(__name__)
 
+
+class SortedRecordSchemaPairs:
```

Comment:
could you add tests?

Reply:
this class is well tested via the `test_case_consistency` tests, are there more cases you would suggest to cover?

Reply:
Nit: maybe rename either the test or add a text in test docstring mentioning which class it is testing